luajit/riscv64.patch

```diff
@@ -28,10 +26,27 @@ index 70afb4b7..141e9c88 100644
  build() {
 -  cd "luajit-2.0-${_commit::7}"
 +  cd "LuaJIT"
+ 
    # Avoid early stripping
    make amalg PREFIX=/usr BUILDMODE=dynamic TARGET_STRIP=" @:"
  }
  
+-check() {
+-  cd "luajit-2.0-${_commit::7}"
+-
+-  # Make sure that _ct was updated
+-  test "${_ct}" == "$(cat .relver)"
+-}
++## FIXME
++## This check won't work in git+source,
++## as $pkgsrc is not a git repo
```

Comment:
Then we may switch back to source tarball instead of cloning from git. I used git for the first patch because we need a cherry-pick which is not the case now. :)

Reply:
Thanks. Can i use something like `https://github.com/infiWang/LuaJIT/archive/172b660f2c0e75d64e85cd9b0df2ae14f1bde8eb.zip`  ?

Reply:
> Can i use something like `https://github.com/infiWang/LuaJIT/archive/172b660f2c0e75d64e85cd9b0df2ae14f1bde8eb.zip` ?

`.tar.gz` is preferred. :)

Reply:
OK. :-)